### PR TITLE
Feature/one key

### PR DESCRIPTION
**Pull request by katenorquay**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Here can explain the process of logging in to the server and testing. Could also have a build file with instructions for how to set up the dev environment.
```
Comment (nigelramsay) [inline DEVELOPMENT.md]: Can you fill this in? Would be very handy for new time.

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  var parsedWidgetOptions = safeParseJSONObject(window.AddressFinderConfig.widgetOptions);
+  var parsedNZWidgetOptions = safeParseJSONObject(window.AddressFinderConfig.nzWidgetOptions);
+  var parsedAUWidgetOptions = safeParseJSONObject(window.AddressFinderConfig.auWidgetOptions);
+
+  window.AF._plugin = new AF.WooCommercePlugin({
+    nzKey: window.AddressFinderConfig.key_nz || window.AddressFinderConfig.key || window.AddressFinderConfig.key_au,
+    auKey: window.AddressFinderConfig.key_au || window.AddressFinderConfig.key || window.AddressFinderConfig.key_nz,
+    nzWidgetOptions: parsedNZWidgetOptions || parsedWidgetOptions || {},
+    auWidgetOptions: parsedAUWidgetOptions || parsedWidgetOptions || {},
+    debug: window.AddressFinderConfig.debug || true
```
Comment (nigelramsay) [inline src/main.js]: change `true` to `false` as we probably only need debug mode when required

File: src/woocommerce_plugin.js [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+import "core-js/fn/array/find"
+import "core-js/fn/array/from"
+import "core-js/fn/object/values"
+
+// The AddressFinder plugin for WooCommerce adds an autocomplete capability to the
+// delivery and billing address fields of your online store.
+//
+// https://wordpress.org/plugins/addressfinder-woo/
+//
+// VERSION: 1.1.2
```
Comment (nigelramsay) [inline src/woocommerce_plugin.js]: version number should match

File: woocommerce-addressfinder.php [hunk 4]
```diff
@@ -52,32 +52,52 @@ function add_addressfinder_widget( $checkout ) {
 	function addressfinder_settings( $settings ) {
 		$settings[] = array( 'name' => __( 'AddressFinder Settings', 'text-domain' ),
 			'type' => 'title',
-			'desc' => __( 'AddressFinder supports New Zealand and Australia. You may enter a key for either, or both.', 'text-domain' ),
+			'desc' => __( 'AddressFinder supports New Zealand and Australia. You only need to enter one licence key to use both widgets', 'text-domain' ),
```
Comment (nigelramsay) [inline woocommerce-addressfinder.php]: Remove the statement 

> You only need to en...

File: woocommerce-addressfinder.php [hunk 5]
```diff
@@ -52,32 +52,52 @@ function add_addressfinder_widget( $checkout ) {
...
-			'desc_tip' => __( 'The Key shown in the AddressFinder portal', 'text-domain' ),
-			'id'       => 'af-key',
-			'type'     => 'text',
-			'desc'     => __( 'Find your AddressFinder Key from <a href="https://portal.addressfinder.io" target="_blank">AddressFinder Portal</a>', 'text-domain' ),
-		);
+		$af_key_nz = esc_attr( get_option( 'af-key' ) );
+		$af_key_au = esc_attr( get_option( 'af-key-au' ) );
+		if($af_key_nz && $af_key_au) {
+			$settings[] = array(
+				'name'     => __( 'Licence Key for New Zealand', 'text-domain' ),
```
Comment (nigelramsay) [inline woocommerce-addressfinder.php]: no need for capital "K" for key

File: woocommerce-addressfinder.php [hunk 6]
```diff
@@ -52,32 +52,52 @@ function add_addressfinder_widget( $checkout ) {
...
+				'desc_tip' => __( 'The Key shown in the AddressFinder portal', 'text-domain' ),
+				'id'       => 'af-key',
+				'type'     => 'text',
+				'desc'     => __( 'Find your AddressFinder Key from <a href="https://portal.addressfinder.io" target="_blank">AddressFinder Portal</a>', 'text-domain' ),
```
Comment (nigelramsay) [inline woocommerce-addressfinder.php]: How about a target="_blank" here

File: src/main.js (hunk 2 above)
Comment (katenorquay) [inline src/main.js]: 👍

File: src/woocommerce_plugin.js (hunk 3 above)
Comment (katenorquay) [inline src/woocommerce_plugin.js]: 👍

File: woocommerce-addressfinder.php (hunk 5 above)
Comment (katenorquay) [inline woocommerce-addressfinder.php]: 👍

File: woocommerce-addressfinder.php (hunk 4 above)
Comment (katenorquay) [inline woocommerce-addressfinder.php]: 👍

File: DEVELOPMENT.md (hunk 1 above)
Comment (katenorquay) [inline DEVELOPMENT.md]: 👍

File: woocommerce-addressfinder.php (hunk 6 above)
Comment (katenorquay) [inline woocommerce-addressfinder.php]: Not sure what you mean with this comment, can we discuss?

**Pull request merged**